### PR TITLE
Add solution to first lab

### PR DESCRIPTION
**Pull request by StoyanYanev**

### CODE REVIEW

File: 62240.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <h1>Стоян Янев, 62240</h1>
+    <p>Студент, 3 курс Софтуерно инженерство</p>
+    <ul>
+        <li>Банан</li>
+        <li>Бадеми</li>
+        <li>Пица</li>
+        <li>Спагети</li>
+        <li>Тирамису</li>
+    </ul>
+    <a href="https://bg-bg.facebook.com/">Facebook</a>
```
Comment (martinely95) [inline 62240.html]: Липсва отварянето на нов екран. https://iliasky.com/www/presentations/INTRO.html?read=1#15 предпоследното условие

File: 62240.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
```
Comment (martinely95) [inline 62240.html:3]: Може да искате да добавите `<meta content="text/html; charset=UTF-8" http-equiv="Content-Type">` в head частта заради кирилицата. Повече информация:

- https://stackoverflow.com/questions/22735993/html5-encoding-cyrillic

- https://www.w3schools.com/charsets/ref_html_utf8.asp

**Review by martinely95 — CHANGES_REQUESTED** [review]

**Comment by martinely95** [comment]
Може да искате да пуснете файла на валидатор: https://validator.w3.org/

File: 62240.html (hunk 2 above)
Comment (martinely95) [reply 62240.html:3]: Furthermore, the meta tag for the charset will enhance your protection from XSS (Cross-Site Scripting) attacks. It is important that the browser is not left alone to guess the charset as some unexpected behaviour may be observed, especially with escaped characters. 
More info: 
https://owasp.org/www-project-top-ten/OWASP_Top_Ten_2017/Top_10-2017_A7-Cross-Site_Scripting_(XSS)  
https://owasp.org/www-project-cheat-sheets/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet
https://softuni.bg/trainings/resources/officedocument/46136/presentation-tech-stage-web-applications-security/2769

File: 62240.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <ul>
+        <li>Банан</li>
+        <li>Бадеми</li>
+        <li>Пица</li>
+        <li>Спагети</li>
+        <li>Тирамису</li>
+    </ul>
+    <a href="https://bg-bg.facebook.com/">Facebook</a>
+    </body>
+</html>
```
Comment (tantanitaa) [inline 62240.html:19]: You can add lang attribute for example.

File: 62240.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    <title>Упражнение 1</title>
+    </head>
+    <body>
+    <h1>Стоян Янев, 62240</h1>
+    <p>Студент, 3 курс Софтуерно инженерство</p>
+    <ul>
+        <li>Банан</li>
+        <li>Бадеми</li>
+        <li>Пица</li>
+        <li>Спагети</li>
```
Comment (martinely95) [inline 62240.html:14]: New cooment

File: 62240.html [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="bg">
+    <head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
+    <title>Упражнение 1</title>
+    </head>
+    <body>
```
Comment (martinely95) [inline 62240.html:7]: Comment 2

**Review by martinely95 — CHANGES_REQUESTED** [review]

**Pull request merged**